### DISCUSSION TITLE
help me

### DISCUSSION THREAD

**Discussion by Justin-Emmanuel** [twbs/bootstrap#38985] [Q&A]
hi,as iam new to frontend developing, i just completed the basics of html and css ,i was here for a reason that iam not able to access the bootstrap 5.3.1 the latest version as i downloaded the file and pasted over a new folder in visual code but iam not able to access, can you please help me out to solve this, please guide me the process! thanking you.

**Comment by ishan-18** [comment]
To use Bootstrap 5.3.1 in your frontend project, you don't need to manually download and paste the files in your project folder. Instead, you can include Bootstrap directly in your HTML file using either a CDN or by installing it through npm.

1. Using CDN (Content Delivery Network):
You can include the Bootstrap CSS and JavaScript files directly in your HTML file using the following code in the head section:

```html
<!DOCTYPE html>
<html>
<head>
  <title>Your Page Title</title>
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet">
</head>
<body>
  <!-- Your HTML content here -->
  
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js"></script>
</body>
</html>
```

2. Using npm (Node Package Manager):
If you prefer to install Bootstrap through npm, follow these steps:

a. Open a terminal in your project folder and run the following command to install Bootstrap:

```bash
npm install bootstrap@5.3.1
```

b. After the installation, include the Bootstrap CSS and JavaScript files in your HTML file like this:

```html
<!DOCTYPE html>
<html>
<head>
  <title>Your Page Title</title>
  <link href="node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
</head>
<body>
  <!-- Your HTML content here -->
  
  <script src="node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
</body>
</html>
```

Now you should be able to access and use Bootstrap 5.3.1 in your frontend project. You can start utilizing Bootstrap classes and components in your HTML to style and enhance your web pages.
